helix-core/src/position.rs

```diff
@@ -73,6 +75,13 @@ pub fn coords_at_pos(text: RopeSlice, pos: usize) -> Position {
 /// Takes \t, double-width characters (CJK) into account as well as text
 /// not in the document in the future.
 /// See [`coords_at_pos`] for an "objective" one.
+///
+/// This function should be used very rarely. Usually `visual_offset_from_anchor`
+/// or `visual_offset_from_block` is preferable. However when you want to compute the
+/// actual visual row/column in the text (not what is actually shown on screen)
+/// then you should use this function. For example aligning text should ignore virtual
+/// text and softwrap.
+#[deprecated = "Doesn't account for softwrap or decorations, use visual_offset_from_anchor instead"]
 pub fn visual_coords_at_pos(text: RopeSlice, pos: usize, tab_width: usize) -> Position {
```

Comment:
Which places still use this?

Reply:
A couple commands in `commands.rs` still use this where it seemed to me that we really care about the column position in the text and not on screen poistion (like alignment). See https://github.com/helix-editor/helix/pull/5420#discussion_r1090075315 for more details.

I want to revisit those functions in the future or potentially add a seperate variant that respects softwrap/virtual text but I wanted to keep the diff smaller here so I left them alone.